src/ConservationChecker.jl

```diff
@@ -16,193 +16,184 @@ using UnPack
 using Plots
 using ClimaAtmos: RRTMGPI
 using ClimaLSM
-using ..Utilities: CoupledSimulation, swap_space!
-using ..Interfacer: SurfaceStub
+using ClimaCoupler.Utilities: CoupledSimulation, swap_space!
+import ClimaCoupler: Interfacer
 
-export AbstractCheck, EnergyConservationCheck, WaterConservationCheck, check_conservation!, plot_global_conservation
+export AbstractConservationCheck,
+    EnergyConservationCheck, WaterConservationCheck, check_conservation!, plot_global_conservation
 
-abstract type AbstractCheck end
+abstract type AbstractConservationCheck end
 
 """
-    EnergyConservationCheck{A} <: AbstractCheck
+    EnergyConservationCheck{A} <: AbstractConservationCheck
 
-Struct of type `AbstractCheck` containing global energy conservation logs.
+Struct of type `AbstractConservationCheck` containing global energy conservation logs.
 """
-struct EnergyConservationCheck{A} <: AbstractCheck
-    ρe_tot_atmos::A
-    ρe_tot_land::A
-    ρe_tot_ocean::A
-    ρe_tot_seaice::A
-    toa_net_source::A
-    ice_base_source::A
+mutable struct EnergyConservationCheck <: AbstractConservationCheck
+    sums::NamedTuple
```

Comment:
Should this be `sims`? or is `sums` correct?

Reply:
it's sums of the sims in fact - originally I had called it fields, but then didn't want it to be confused with `ClimaCore.Fields.Field`s.